### DISCUSSION TITLE
Our community has been renamed to the zopen community

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#866] [Announcements]
You can find out website at http://zopen.community/ and our organization at https://github.com/orgs/zopencommunity

**Comment by v1gnesh** [comment]
Many (or at least some) sites do wholesale block of "suspicious" top-level domains.
So please consider this a report of the first such case where I can't get  to the website at all from work :)

**Reply by IgorTodorovskiIBM** [reply]
Is your company able to add .community to their allowlist? We were told we're not the only Linux foundation community with the .community tld

**Reply by v1gnesh** [reply]
No, unfortunately. In more than one site, for sure, exceptions are reviewed per requester. 
Any site that uses filter lists in their proxy are susceptible to this.
The proxy vendor may choose to use filter list(s) that just block whole TLDs.

**Reply by IgorTodorovskiIBM** [reply]
Not sure if this helps, but here's a few other LF .community projects:

https://openhpc.community/
https://chaoss.community/

Some others...
blender.community
github.community
openresearch.community

**Comment by lnlyssg** [comment]
Any chance of putting in some redirects? All the top hits on Google are leading to dead links which makes it very hard to find the project now!

**Reply by IgorTodorovskiIBM** [reply]
Thanks, actually we were waiting for the 30 day period to expire so that we can reclaim the old organization name. But, I tried today and was able to claim it. 

I created a README with the link to the new community. I don't think there's a way to automatically redirect

**Reply by IgorTodorovskiIBM** [reply]
https://github.com/ZOSOpenTools
